recipe/install_connexion.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+$PYTHON -m pip install . -vv --no-deps --no-build-isolation
```

Comment:
I think the issue is that inside the output block environment variables are lost.
My guess is that if there were a jinja2 variable constructed from the environment variable at the first then it would work properly.
I am not sure if the following would do that or not.

{% set python_bin = {{ PYTHON }} %}

Reply:
I don't think that would help but maybe.  I've always seen this script apprach.